portfolio/src/main/webapp/calendar.js

```diff
@@ -0,0 +1,50 @@
+class CalendarEvent{
+  constructor(name, startTime, endTime) {
```

Comment:
Let's indicate the time unit in the variable names.

ex: startTimeMillis, endTimeMillis

Also, you may want to start thinking about time zone, so that you don't have to do a huge refactor later. 

One approach you can consider is using the Date Javascript objects, as opposed to these timestamps. If we pass around the timestamps, then it can be difficult to reason about time zone, time unit, etc.

Reply:
Updated. Now I am using the Date object, to build startTime and endTime.